bluebinder_post.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+if [ "$(getprop persist.vendor.service.bdroid.bdaddr)" != "" ]; then
+    mkdir -p /var/lib/bluetooth
+    echo $(getprop persist.vendor.service.bdroid.bdaddr |awk -F: '{do printf "%s"(NF>1?FS:RS),$NF;while(--NF)}') > /var/lib/bluetooth/board-address
+    chown root:root /var/lib/bluetooth/board-address
+    chmod 644 /var/lib/bluetooth/board-address
+    exit 0
+fi
+
+exit 0
```

Comment:
I think we should check here if /var/lib/bluetooth/board-address was generated, and if it was not then exit 1 instead